packages/client/hmi-client/src/services/search.ts

```diff
@@ -0,0 +1,7 @@
+import API from '@/api/api';
+import { AssetType } from '@/types/Types';
+
+export async function search(text: string, assetType: AssetType) {
+	const response = await API.get(`/search-by-asset-type/${assetType}?text=${text}`);
+	return response?.data ?? [];
+}
```

Comment:
Maybe a more specific name so things are less confusing down the road? "searchAsset"?

Does `text` need to be sanitized, url-encoded, escaped and whatnot? Not sure how we are processing it at the API level...

Reply:
Maybe a `searchByAssetType`?